test/src/e2e_vm_tests/test_programs/should_fail/shadow_import/src/main.sw

```diff
@@ -8,7 +8,7 @@ use ::bar::{Bar1, Bar1 as Foo};
 // This is not okay because symbol `Foo` is already reserved
 use ::bar::Bar2 as Foo;
 
-// This is not okay 
+// This is not okay
 use ::bar::{Bar2, Bar2};
 
 // This not okay now because Bar1 and Bar2 have already been imported
```

Comment:
I see you've already removed the checks from the `test.toml`, but perhaps we should remove this line (and the one under it) too? Otherwise it could be a bit confusing to come across for future maintainers.

Great to see we can now shadow glob imports :partying_face: